lib/handbook/addon/docs/components/tags-widget/editable/component.ts

```diff
@@ -0,0 +1,17 @@
+import { action } from '@ember-decorators/object';
+import Component from '@ember/component';
+
+export default class Editable extends Component.extend() {
+    // required arguments
+    tags: string[] = ['Tag 1', 'Tag 2', 'Tag 3'];
+
+    @action
+    _addTag(this: Editable, tag: string) {
+        this.set('tags', [...this.tags, tag].sort());
+    }
```

Comment:
Instead of defining these actions for each demo component, you should be able to define them one in a controller. See: 
https://github.com/CenterForOpenScience/ember-osf-web/blob/15ffd09f1c8afc2293808f0fda923b552acc5b66/lib/handbook/addon/docs/components/delete-button/controller.ts#L11-L15
and:
https://github.com/CenterForOpenScience/ember-osf-web/blob/develop/lib/handbook/addon/docs/components/delete-button/template.md

and:
https://github.com/CenterForOpenScience/ember-osf-web/blob/15ffd09f1c8afc2293808f0fda923b552acc5b66/lib/handbook/addon/docs/components/delete-button/demo-default/template.hbs#L2-L6